### DISCUSSION TITLE
Remove background blur

### DISCUSSION THREAD

**Discussion by nick-0101** [enesozturk/react-native-hold-menu#51] [Ideas]
I think it would be nice to include a prop like `blurBackdrop={true || false}`. I feel this would be a good addition because for some use cases, the background blur is uneeded.

**Comment by nick-0101** [comment]
I've forked this branch and added `disableBackdrop={true || false}` which removes the blurred backdrop if set to true, and keeps it if set to false. I've created a pull request, hopefully it gets merged. In the meantime I've uploaded the forked version to npm [https://www.npmjs.com/package/react-native-hold-menu-forked](https://www.npmjs.com/package/react-native-hold-menu-forked)

**Reply by enesozturk** [reply]
Hi @phoenixbeats01, thanks for the contribution but you need to open PR to by repo. After opening it I will review the changes and may merge them.

**Comment by hadnet** [comment]
I created a package for React Native CLI (no Expo) that allows disabling the backdrop blur: https://www.npmjs.com/package/@hadnet/react-native-hold-menu-no-expo
